web/components/issues/issue-layouts/calendar/calendar.tsx

```diff
@@ -129,6 +160,42 @@ export const CalendarChart: React.FC<Props> = observer((props) => {
               />
             )}
           </div>
+
+          {/* mobile view */}
+          <>
+            <div className="md:hidden">
+              <p className="p-4 text-xl font-semibold">
+                {`${selectedDate.getDate()} ${
+                  MONTHS_LIST[selectedDate.getMonth() + 1].title
+                }, ${selectedDate.getFullYear()}`}
+              </p>
+              {issueIdList &&
+                issueIdList?.length > 0 &&
+                issueIdList?.map((issueId) => {
+                  if (!issues?.[issueId]) return null;
```

Comment:
@1akhanBaheti Please create a new component for issue inside the list of issues. Since this is issues, destructuring in the parent component might cause the entire list to re render.i.e., don't use the issue.issueId in where you are mapping through the issues. Do it in the individual issue component.